roles/validate/tasks/verify_ndfc_authorization.yml

```diff
@@ -21,6 +21,13 @@
       # no_log: true
       delegate_to: localhost
       # when: ansible_user is defined and ansible_password is defined
+  
+    - name: Get Cisco NDFC Version
```

Comment:
Could we move this into the `common` role instead of `validate`?  Want to try to move any NDFC connection points outside of `validate`

Reply:
I moved this to a new connectivity check role and made it a dependency to create.